Provider fails to compile on Windows

```
gyro-aws-provider\src\main\java\gyro\aws\elbv2\FixedResponseAction.java:33: error: unmappable character (0x9D) for encoding windows-1252
 *         type: “fixed-response�?
                                    ^
```

--- a/src/main/java/gyro/aws/elbv2/FixedResponseAction.java
+++ b/src/main/java/gyro/aws/elbv2/FixedResponseAction.java
@@ -30,7 +30,7 @@ import software.amazon.awssdk.services.elasticloadbalancingv2.model.FixedRespons
  * .. code-block:: gyro
  *
  *     action
- *         type: “fixed-response”
+ *         type: "fixed-response"
  *
  *         fixed-response-action
  *             content-type: "text/plain"

